src/webpack.js

```diff
@@ -74,11 +89,63 @@ const getBuildObjectFromBundle = callback => {
 };
 
 /**
- * @desc Watching webpack build process
+ * @desc Watching and Streamming webpack build process
  * @param {bool} devMode return true when arg is --dev
  * @param {function} callback return data
  */
-const watching = (devMode, callback) => {
+const watchAndStreamingData = (devMode, callback) => {
+  watchingChanges(devMode, () => {
+    getBuildObjectFromBundle(callback);
+  });
+};
+
+/**
+ * @desc Write bundle file
+ * @param {bool} devMode return true when arg is --dev
+ */
+const writeBundleFile = devMode => {
+  watchingChanges(
+    devMode,
+    () => {
+      writeBundleBuildFile();
+    },
+    true, //stop watching
+  );
+};
+
+/**
+ * @desc Write bundle build file
+ */
+const writeBundleBuildFile = () => {
```

Comment:
this function should only be responsible for writing data based on the name. The name is miss leading.